runtime/crab/src/pallets/evm.rs

```diff
@@ -176,24 +176,21 @@ impl DispatchValidateT<AccountId, RuntimeCall> for DarwiniaDispatchValidator {
 		origin: &AccountId,
 		call: &RuntimeCall,
 	) -> Option<fp_evm::PrecompileFailure> {
-		if let Some(root) = pallet_sudo::Pallet::<Runtime>::key() {
```

Comment:
This would mean no way to call dispatch using ROOT, and it also depends whether their are still some contracts depending on sudo key?